pull_request_template.md

```diff
@@ -1,6 +1,6 @@
-## Issue and/or context:
+**Issue and/or context:**
```

Comment:
I use `####` when I edit them.  Appears to be equivalent.

Reply:
@eddelbuettel that would solve only one of the two problems I noted in the description, namely:

```
#### Issue: foo
```

is still a mistake one needs to immediately re-edit to

```
#### Issue:

foo
```

I want to make this less error-prone and less (micro-)time-consuming.